Framework/SINQ/src/PoldiPeakSearch.cpp

```diff
@@ -611,12 +612,12 @@ void PoldiPeakSearch::exec() {
    */
   std::vector<PoldiPeak_sptr> peakCoordinates =
```

Comment:
Get result by reference `std::vector<PoldiPeak_sptr> &peakCoordinates`, I think it can also be const? Also could do `const auto &peakCoordinates`